src/docs/product/issues/issue-details/performance-issues/regressions.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: "Function Regression"
+sidebar_order: 40
+description: "Learn more about Function Regression Issue and how to diagnose and fix them."
+---
+
+Function Regression issues capture a generic class of problems where the duration of a function has increased over time. This can lead to degraded performance in your application. Sentry attempts to monitor some of your most common functions out of the box and report any possible regressions under the Function Regression issue type.
+
+## Detection Criteria
+
+[Profiling](/product/profiling) must be enabled for Sentry to detect Function Regression issues. Once set up, Sentry will periodically check the 95th percentile function duration of the most common application functions in your project. When a significant change in the function duration for a function is detected and it has sustained for some time, a regression issue is reported.
+
+## Charts
+
+The chart shows a window of up to 14 days before and after where the regression was identified. Here you can see the p95 function duration as well as the function throughput.
+
+A common cause of regressions is an increase in throughput of a function in the code path that contains a mutex which is now causing lock contention.
+
+## Function Evidence
+
+To find additional information about your regressed function, go to the "Function Evidence" section which shows the following:
+
+- **Function Name:** The name of the function that has regressed.
+- **Package Name:** The package of the function that has regressed.
+- **File Name:** The file name of the function that has regressed.
+- **Regression Date:** The date and time at which the regression started.
+
+## Transactions Impacted
+
+The "Transactions Impacted" section shows a list of transactions where this regression may be observed. Next to each transaction, you can find the following:
+
+- A sparkline showing the 95th percentile function duration of the regressed function within the transaction.
```

Comment:
```suggestion
- A sparkline showing the p95 function duration of the regressed function within the transaction.
```

I feel like p95 is pretty common and doesn't need to be spelled out after the first mention